### ISSUE TITLE
Required to forcefully kill program after recording demo

### ISSUE THREAD

**Issue by Overv** [faressoft/terminalizer#41]
I just recorded my first demo with `terminalizer record demo` and ended it with Ctrl-D. However, after that it shows the following message:

```
Open the following link in your browser and login into your account
https://terminalizer.com/token?token=xxx

When you do it, press any key to continue
Enjoy !
```

I pressed enter to skip it because I have no desire to create an account, but it seems to keep coming back and Ctrl-C doesn't exit out of it. I had to use `kill -9 <pid>` to exit the program. Is there a global option to disable this "feature"?

**Comment by riywo** [comment]
Probably we need an option to disable these lines?

https://github.com/faressoft/terminalizer/blob/516bb97726a0fab417d5374c3aa24be7c08f7cae/commands/record.js#L162-L165

**Comment by faressoft** [comment]
Will be released with v0.6.1.

**Comment by crescentrose** [comment]
Sorry if I missed it, but what's the configuration option to disable automatic share after recording again?
